### DISCUSSION TITLE
Does nnUNetv2 have built-in function to produce performance metrics on a test set?

### DISCUSSION THREAD

**Discussion by nicholasjprimianomd** [MIC-DKFZ/nnUNet#2067] [Q&A]
Does nnUNetv2 have built-in function to produce performance metrics on a test set?

**Comment by aspil** [comment]
After you install the project you can use the `nnUNetv2_evaluate_folder` or `nnUNetv2_evaluate_simple` commands, they are slightly mentioned at the end of [ignore_label.md](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/ignore_label.md)

Check the usage using `nnUNetv2_evaluate_folder -h`. Their function entrypoints are in [https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunetv2/evaluation/evaluate_predictions.py](https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunetv2/evaluation/evaluate_predictions.py)
